README.rst

```diff
@@ -14,21 +15,49 @@ openwisp-network-topology
 .. image:: https://badge.fury.io/py/openwisp-network-topology.svg
     :target: http://badge.fury.io/py/openwisp-network-topology
 
-------------
-
-OpenWISP 2 network topology module (built using Python and Django web-framework).
-
-.. figure:: docs/images/visualizer.png
-   :align: center
-   :alt:
-
-------------
+.. image:: docs/demo_network_topology.gif
 
 .. contents:: **Table of Contents**:
    :backlinks: none
    :depth: 3
 
-------------
+Current features
+----------------
+
+* **network topology collector** supporting different formats:
+    - NetJSON NetworkGraph
+    - OLSR (jsoninfo/txtinfo)
+    - batman-adv (jsondoc/txtinfo)
+    - BMX6 (q6m)
+    - CNML 1.0
+    - OpenVPN
+    - additional formats can be added by `specifying custom parsers <#netjsongraph-parsers>`_
+* **network topology visualizer** based on `netjsongraph.js <https://github.com/openwisp/netjsongraph.js>`_
+* **simple HTTP API** that exposes data in `NetJSON <http://netjson.org>`__ *NetworkGraph* format
+* **admin interface** that allows to easily manage, audit, visualize and debug topologies and their relative data (nodes, links)
+* **receive topology** from multiple nodes
+* **topology history**: allows saving daily snapshots of each topology that can be viewed in the frontend
+
+Project goals
+-------------
+
+* make it easy to visualize network topology data for the formats supported by `netdiff <https://github.com/openwisp/netdiff>`_
+* expose topology data via RESTful resources in *NetJSON NetworkGraph* format
+* make it easy to integrate in larger django projects to improve reusability
+* make it easy to extend its models by providing abstract models (**needs improvement in this point**)
```

Comment:
I think we can remove "needs improvement in this point"

Reply:
Lol, yes, I'll remove that! :smile: